### ISSUE TITLE
Support rendering for LimeSeg

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/sciview#193]
Linking to https://github.com/NicoKiaru/LimeSeg/issues/6

**Comment by kephale** [comment]
@NicoKiaru I'm bringing this up again because we have a hackathon going this week. Would LimeSeg be a good demo for 1.0.0?

**Comment by NicoKiaru** [comment]
Man. I wish... Maybe I've got a bit of time to look at it mid July. I need to have sciview working in intellij. Is there a simple kind of sciview scijava plugin demo project that I can clone and test ?

**Comment by kephale** [comment]
Well @NicoKiaru its almost July 🤣 I'll just link the example project while i'm here: https://github.com/scenerygraphics/minimal-sciview-example-project

**Comment by NicoKiaru** [comment]
Awesome!

I don't know when I've have time to check this, but at least I wanted to give a short try at building it.
* The CI from the template fails when opened in my repo: https://github.com/NicoKiaru/my-first-sciview-project
* And I had to choose between gradle and maven when opening in intellij:
![image](https://github.com/scenerygraphics/sciview/assets/20223054/57bd7409-e3ba-4401-9b8e-e4734b322bf2)

I chose gradle. I'll let you know how that goes.

**Comment by NicoKiaru** [comment]
i think I need a jdk21 because of this message:
```
C:\Users\chiarutt\Dropbox\BIOP\my-first-sciview-project\src\main\java\sc\iview\minimal\MyDemo.java:31: error: cannot access Sphere
import graphics.scenery.Sphere;
                       ^
  bad class file: C:\Users\chiarutt\.gradle\caches\modules-2\files-2.1\graphics.scenery\scenery\0.11.0\8aea34f37cc2e9aee5b856cb890258e94f844846\scenery-0.11.0.jar(/graphics/scenery/Sphere.class)
    class file has wrong version 65.0, should be 60.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

And I've set it as the jdk to choose in intellij, but it still seems to give errors.

**Comment by NicoKiaru** [comment]
Yep, I think so as well, the problem I guess is that I'm setting up the jdk in intellij, but gradle is probably ignoring that.

![image](https://github.com/scenerygraphics/sciview/assets/20223054/1d7638f6-be1c-4a02-a62a-f272dbff6590)

**Comment by skalarproduktraum** [comment]
@NicoKiaru Your guess is correct - but the culprit is not Gradle, but the Powershell you're running it from. You'd need to set the environment variable JAVA_HOME to the Java 21 JDK, then run `./gradlew`. That ought to solve your issue 👍

**Comment by skalarproduktraum** [comment]
(Side note: When you install a new JDK on Windows, the installer usually gives you the option to set JAVA_HOME globally as well)

**Comment by NicoKiaru** [comment]
I'm trying this:

![image](https://github.com/scenerygraphics/sciview/assets/20223054/fd1c5b83-005b-4c07-b1e0-cc3b6fe5e5a6)

because for some reason I can't make the shell swallow anythin env variable. I don't know why and this is driving me crazy

**Comment by NicoKiaru** [comment]
Tadaa!!!

![image](https://github.com/scenerygraphics/sciview/assets/20223054/5d7e1dec-3a78-411b-ad64-f9ff73ae41a8)

**Comment by skalarproduktraum** [comment]
Apparently, in Powershell, you do `$env:JAVA_HOME="C:\path\to\jdk"`. But great to see it works for you from IntelliJ 👍

**Comment by NicoKiaru** [comment]
Any way I can run things with the LegacyService ? I'm getting this message:
```
Caused by: java.lang.RuntimeException: No _hooks field found in ij.IJ
```

I also tried to add
```
static {
        net.imagej.patcher.LegacyInjector.preinit();
    }
```

but it didn't solve the issue

**Comment by NicoKiaru** [comment]
I added this in vm options and now it works fine:

 "--add-opens=java.base/java.lang=ALL-UNNAMED",
                "--add-opens=java.base/java.lang.invoke=ALL-UNNAMED",
                "--add-opens=java.base/java.net=ALL-UNNAMED",
                "--add-opens=java.base/java.nio=ALL-UNNAMED",
                "--add-opens=java.base/java.time=ALL-UNNAMED",
                "--add-opens=java.base/java.util=ALL-UNNAMED",
                "--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED",
                "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
                "--add-opens=java.base/sun.util.calendar=ALL-UNNAMED"

**Comment by NicoKiaru** [comment]
Also: how can I choose the GPU used for sciview ? I have an integrated graphics card and a nvidia gpu, and I d like to use the nvidia card.

**Comment by NicoKiaru** [comment]
I occasionally get this message:

Caused by: java.lang.IllegalArgumentException: Color parameter outside of expected range: Red Green Blue

**Comment by NicoKiaru** [comment]
It looks like the commands that I define in the repo are not added into Fiji's menu. I'll need to confirm though

**Comment by NicoKiaru** [comment]
Last feedback from today: I couldn't get a SourceAndConverter to be displayed. I tried in BigVolumeViewer in the same repo, but I get this error:

Caused by: java.lang.ClassNotFoundException: com.jogamp.opengl.GLEventListener
